extensions/ql-vscode/media/dark/github.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Consider an HTML comment (if SVG allows them) or a README file stating where you got these icons from.